hooks/useFcm.ts

```
@@ -0,0 +1,107 @@
+      listener.current = null;
+      listener.current = Notifications.addNotificationReceivedListener(n =>
+        withDedup(n.request.identifier, ()=>{}),
+      );
+    }
+
+    if(!clicker.current) {
+      clicker.current = null;
+      clicker.current = Notifications.addNotificationResponseReceivedListener(r => {
+        const url = convertAction(r);
+        if (url && url !== 'NO_ACTION') {
+              setNotificationList(prev => 
```

ESLint

[error] Replace `··············setNotificationList(prev·=>·` with `··········setNotificationList((prev)·=>`

(prettier/prettier)